### DISCUSSION TITLE
Can the mass private feature be used for unprivating?

### DISCUSSION THREAD

**Discussion by pooperoo** [AprilSylph/XKit-Rewritten#1365] [Help Desk]
Or is there any other feature that can be used to mass unprivate? Thanks!

**Comment by AprilSylph** [comment]
Unfortunately no. Mass Privater only exists because Tumblr's Mass Post Editor has the option to private posts, which allows us to leverage its API. Since the Mass Post Editor doesn't have an option to unprivate posts in bulk, we also don't have a Mass Unprivater.
